cmd/check.go

```diff
@@ -89,6 +162,8 @@ and/or users.`,
 	flags.StringSliceVar(&checkflags.AuditPlan, "audit-plan", defaultAuditPlan, "audit plan is the ordered list of operator test functions to be called during a capability audit.")
 	flags.StringSliceVar(&checkflags.Packages, "packages", []string{}, "a list of package(s) which limits audits and/or other flag(s) output")
 	flags.BoolVar(&checkflags.AllInstallModes, "all-installmodes", false, "when set, all install modes supported by an operator will be tested")
+	flags.StringVar(&checkflags.ExtraCRDirectory, "extra-cr-directory", "",
+		"directory containing the additional Custom Resources to be deployed by the OperandInstall audit. The manifest files should be located in subdirectories named after the packages they are corresponding to.")
```

Comment:
Can this be specified multiple times?

Reply:
No it should not. You specify for instance `--extra-cr-directory custom_resources_directory/` and the following structure is expected:
```
custom_resources_directory/
├── package_name1
│   ├── manifest_file1.json
│   └── manifest_file2.yaml
└── package_name2
    ├── manifest_file1.json
    └── manifest_file2.yaml
```

So you set only one `--extra-cr-directory` in the CLI, but you can target multiple packages.